ocaml/xapi/import.ml

```diff
@@ -1446,9 +1447,7 @@ let handle_all __context config rpc session_id (xs: obj list) =
 
 (** Read the next file in the archive as xml *)
 let read_xml hdr fd =
-  let xml_string = Bigbuffer.make () in
-  really_read_bigbuffer fd xml_string hdr.Tar_unix.Header.file_size;
-  Xml.parse_bigbuffer xml_string
+  Unixext.really_read_string fd (Int64.to_int hdr.Tar_unix.Header.file_size)
```

Comment:
This could be renamed to `read_next_file`, or we could add a `Xmlrpc.of_string` to the end.

Reply:
There was a reason I did not add the Xmlrpc.to_string but I think that woth the latest changes that reason is no longer there. I’ll fix it or let you know the reason :)

Reply:
Found it. The old Xva module uses a direct approach manipulating and marshalling/unmarshalling the xml. The `read_xml` function is used both by our new `Xmlrpc` and that old `Xml.parse_String |> Xva.from_xml`. Changing that is invasive and out of scope for this pr